_blog/why-null-models-crispr.md

```diff
@@ -0,0 +1,83 @@
+---
+title: What's the point of a simple theory?
+subtitle: A story about the power of null models to shape how we interpret data
+#status: inactive #active if you want it on the frontpage
+
+category: paper
+
+description: |
+  A story about the power of null models to shape how we interpret data.
+
+people: # add people that are involved in this project
+  - maddy
+
+layout: blog # do not change this
+image: '/img/posts/why-null-models-crispr/banfield_data_theory.svg'
+#link:
+no-link: false # if you don't want it's own webpage
+last-updated: 2022-01-08
+# do NOT add 0 (such as 06 for June). Will break everything lol
+# take out the underscore in the name.
+---
+
+## Why do we need theory?
+
+Experimentalists wade through the murky waters of biology, hoping to tease out knowledge by controlling the things they can control and carefully observing what they cannot. In a well-designed experiment, the observed data reveals some truth, and this truth is reached by painstakingly ruling out other explanations. This might sound logical and sytematic, but it's an art form: how do you know when you're done considering reasonable alternatives? And in biology, where some [truly unreasonable](https://www.theatlantic.com/science/archive/2017/06/jumping-spiders-can-see-the-moon/529329/) things can happen, how do you know what's 'reasonable' anyway?
```

Comment:
First sentence is great! I think that the second sentence can be a bit more precise though. I also think it can summarize the main message of your blog: there are often many possible explanations for a given dataset, and theory can help identify them or suss them out. 

So for example, maybe your second sentence could go something like this: In a well-designed experiment, the observed data reveals insight by agreeing with a possible explanation and ruling out others. Or something like that (you can probably make it more elegant haha). I guess the sentence as is all ready says something like this, but when I read _truth_ I don’t know if you mean absolute truth or “truth”. It was only after re-reading the blog that I realized that the “truth” here is a possible explanation, but you’re trying to make the point that there could be others. 

You could also drop “but it’s an art form:”, but that’s just a personal taste.

Reply:
Thanks! Made some edits to the intro to be more clear.